docs/security/upgrading/upgrade-to-ubuntu-16-04.md

```diff
@@ -98,7 +98,7 @@ Remember to perform these steps in a Lish or Glish session:
 
 3.  You're now ready to begin the upgrade to Ubuntu 16.04 LTS:
 
-        do-release-upgrade -d
+        do-release-upgrade
```

Comment:
Hi @navjotjsingh,
I tried running the upgrade without the -d flag like you're suggesting, but I got an error back.
Was there something else you tried here that worked better than what we've got?


Thanks,
Edward